src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -81,15 +85,23 @@ function assembleArrow(
         transformStyle += `translateX(-50%) rotate(${arrowPos === 'top' ? 225 : 45}deg)`;
     }
 
-    const borderStyle = `${borderColor} solid 1px;`;
+    const borderStyle = `${borderColor} solid ${borderWidth}px;`;
     const styleCss = [
-        'position:absolute;width:10px;height:10px;',
+        `position:absolute;width:${arrowSize}px;height:${arrowSize}px;`,
         `${positionStyle};${transformStyle};`,
         `border-bottom:${borderStyle}`,
         `border-right:${borderStyle}`,
-        `background-color:${backgroundColor};`,
-        'box-shadow:8px 8px 16px -3px #000;'
+        `background-color:${backgroundColor};`
     ];
+
+    const shadowBlur = tooltipModel.get('shadowBlur');
+    const shadowColor = tooltipModel.get('shadowColor');
+    const shadowOffsetX = tooltipModel.get('shadowOffsetX');
+    const shadowOffsetY = tooltipModel.get('shadowOffsetY');
+    const boxShadow = `${shadowOffsetX}px ${shadowOffsetY}px ${shadowBlur}px ${shadowColor}`;
+
+    styleCss.push('box-shadow:' + boxShadow);
```

Comment:
If we add `box-shadow` for the arrow, it will look unexpectedly obvious. I'm not sure whether we should disable shadow of arrow.

Reply:
So, should I remove the shadow?

Reply:
Please @pissang @susiwen8 take a look.

Reply:
I remember we dicussed it before and I prefereed not having shadow here. I'm not sure if there is a better solution.

Reply:
Perhaps we need CSS `clip-path` like described in https://stackoverflow.com/questions/43863785/css-arrow-with-border-add-box-shadow/43863886

Reply:
`clip-path` should work. However, it seems to have poor compatibility.

Reply:
@plainheart https://developer.mozilla.org/zh-CN/docs/Web/CSS/clip-path Major browsers except for IE already support it well. It is acceptable having some small issue on IE to have a better experience on other major browsers IMO. Of course, it will be perfect to have other solutions both keep compatibility, simple implementation(no too much code), and good result.

Reply:
One quick question, how am I supposed to use pseudo-elements in inline style? Is there any way or linked stylesheet so that I can add them there?
Thanks

Reply:
@g7i I don't think it's possible to use pseudo-elements in inline style. So the top solution in my StackOverflow link can't be used in our case. Instead the second solution which uses `clip-path` is possible here. We can apply `clip-path` on our arrow element.